app/views/guidances/admin_new.html.erb

```
@@ -1,99 +1,64 @@
-							</div>
-							<div class="div_right_icon_g">
-								<%= link_to( image_tag('help_button.png'), '#', "data-toggle": "popover", rel: "popover", 'data-html' => "true", 'data-content' => _('Select which group this guidance relates to.'))%>
-							</div>
-							<div class="clearfix"></div>
-
-						</td>
-					</tr>
-				</table>
-
-			<!-- submit buttons -->
-
-				<div class="move_2_right">
-					<a id="new_guidance_submit" data-toggle="modal" href="#new_guidance_alert_dialog" class="btn btn-primary"><%= _('Save')%></a>
-
-					<%= link_to _('Cancel'), :back, class: 'btn cancel' %>
-				</div>
-
-			<br />
-			 <%= tinymce :content_css => asset_path('application.css') %>
-		<%end%>
-	</div>
+<div class="content">
+  <%= form_for :guidance, url: {action: 'admin_create'}, html: {id: 'edit_guidance_form', class: 'roadmap-form bordered'} do |f| %>
+    <fieldset class="side-by-side">
+      <div class="form-input">
+        <%= f.label :text %>
+        <%= text_area_tag("guidance-text", "", class: "tinymce") %>
+        <div class="inline">
+          <a href="#" data-toggle="popover" rel: "popover" data-html: "true"
+             data-content="<%= _('Enter your guidance here. You can include links where needed.') %>">
+          <span class="fa fa-question-circle"></span></a>
+        </div>
+      </div>
+      <div class="form-input">
+        <%= f.label _('Themes'), for: :theme_ids %>
+        <%= f.collection_select(:theme_ids, @themes,
+                  :id, :title, {prompt: false, include_blank: 'None'}, {multiple: true}) %>
+        <div class="inline">
+          <a href="#" data-toggle="popover" rel: "popover" data-html: "true"
+             data-content="<%= _('Select which theme(s) this guidance relates to.') %>">
+          <span class="fa fa-question-circle"></span></a>
+        </div>
+      </div>
+      <div class="form-input">
+        <%= f.label :published %>

```

There is no need to translate to standard html, as I said the other day, FormHelper can help as to detect any syntax error whereas using standard html could fail silently.

<%= f.label(:published, _('Published: ') %> should work. Reference at: http://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormHelper.html#method-i-label